qlty-check/src/tool/php.rs

```diff
@@ -169,7 +172,7 @@ impl Tool for PhpPackage {
         let mut env = HashMap::new();
         env.insert(
             "COMPOSER_VENDOR_DIR".to_string(),
-            path_to_native_string(PathBuf::from(format!("{}/vendor", self.directory()))),
+            path_to_native_string(self.workspace_root.join("vendor")),
```

Comment:
I notice that we seem to construct COMPOSER_VENDOR_DIR here and also on line 79 in `composer.rs` with the same logic. Are these producing the same values?

If so, and the values need to be in sync, we should only compute the value once to avoid a bug if one location changes.

Reply:
Actually I think this is no longer needed anymore, since we are going to be installing packages in the `project_root/vendor anyway`. 
This was here before to tell the plugin the vendor path during invocation runs which was previously in the tool install.